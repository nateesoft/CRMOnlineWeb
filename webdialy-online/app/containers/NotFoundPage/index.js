/**
 * NotFoundPage
 *
 * This is the page we show when the user visits a url that doesn't have a route
 *
 */

import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import notfoundPageImages from './404_notfound.png';

const useStyles = makeStyles(() => ({
  root: {
    border: 0,
  },
}));

export default function NotFound() {
  const classes = useStyles();

  return (
    <img
      src={notfoundPageImages}
      width="100%"
      className={classes.root}
      alt=""
    />
  );
}
