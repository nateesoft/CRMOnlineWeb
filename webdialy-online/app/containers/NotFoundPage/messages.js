/*
 * NotFoundPage Messages
 *
 * This contains all the text for the NotFoundPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.NotFoundPage';

export default defineMessages({
  status: {
    id: `${scope}.status`,
    defaultMessage: '404',
  },
  header: {
    id: `${scope}.header`,
    defaultMessage: 'Page not found',
  },
  detail: {
    id: `${scope}.detail`,
    defaultMessage: `Sorry, we couldn't find this page.`,
  },
  buttonHome: {
    id: `${scope}.buttonHome`,
    defaultMessage: `Go to homepage`,
  },
});
