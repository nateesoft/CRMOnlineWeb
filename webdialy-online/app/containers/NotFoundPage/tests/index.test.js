import React from 'react';
import renderer from 'react-test-renderer';

import NotFoundPage from '../index';

describe('NotFoundPage components', () => {
  it('test snapshot', () => {
    const component = renderer.create(<NotFoundPage />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
