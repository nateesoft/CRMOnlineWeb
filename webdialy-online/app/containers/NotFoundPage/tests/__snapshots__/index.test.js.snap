// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render and match the snapshot 1`] = `
<img
  alt=""
  class="makeStyles-root-1"
  src="IMAGE_MOCK"
  width="100%"
/>
`;
